webapp/channels/src/components/menu/menu.tsx

```diff
@@ -203,21 +201,13 @@ export function Menu(props: Props) {
 
         if (props.menuButtonTooltip && props.menuButtonTooltip.text && !isMobileView) {
             return (
-                <OverlayTrigger
-                    delayShow={OVERLAY_TIME_DELAY}
+                <WithTooltip
+                    id={props.menuButtonTooltip.id}
+                    title={props.menuButtonTooltip.text}
                     placement={props?.menuButtonTooltip?.placement ?? 'top'}
-                    overlay={
-                        <Tooltip
-                            id={props.menuButtonTooltip.id}
-                            className={props.menuButtonTooltip?.class ?? ''}
-                        >
-                            {props.menuButtonTooltip.text}
-                        </Tooltip>
-                    }
-                    disabled={isMenuOpen}
```

Comment:
We didnt add disabled back here

Reply:
@larkox can you please verify the disabled prop behavior do we need that?

Reply:
@M-ZubairAhmed  I have added the disabled

Reply:
@Camillarhi CI is still failing can you check once again